### ISSUE TITLE
Adding Custom URLS

### ISSUE THREAD

**Issue by rajsodhinyu** [ossd-s24/studyTool#4]
I think being able to add custom URLs to this extension would be a good idea. As per the contribution guidelines, I discussed the idea with @hminsky2002 .

**Comment by hminsky2002** [comment]
This would be fantastic! Approved.

**Comment by benzoms** [comment]
I am also interested in this, could I give it a go? Maybe to start, only custom urls in the form of youtube embeddings.

**Comment by hminsky2002** [comment]
> I am also interested in this, could I give it a go? Maybe to start, only custom urls in the form of youtube embeddings.

Absolutely! Thank you for the interest. Feel free to make a PR and I'll review it asap!
